packages/atomic/src/locales.json

```diff
@@ -3,6 +3,14 @@
     "en": "Search",
     "fr": "Recherche"
   },
+  "searchBox": {
+    "en": "Insert a query. Press enter to send.",
+    "fr": ""
```

Comment:
I think if you look in the JSUI string file, you have the french for that:
`Insérer une requête. Appuyez sur Entrée pour envoyer.